2014-05-12 18:15:39 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [C:\xampp\htdocs\cake\lib\Cake\Model\Datasource\DboSource.php, line 458]
2014-05-12 18:15:39 Error: [FatalErrorException] Maximum execution time of 30 seconds exceeded
Request URL: /shop/products
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Maximum executi...', 'C:\xampp\htdocs...', 458)
#1 [internal function]: ErrorHandler::handleError(1, 'Maximum executi...', 'C:\xampp\htdocs...', 458, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Maximum executi...', 'C:\xampp\htdocs...', 458, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-12 22:13:29 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [C:\xampp\htdocs\cake\lib\Cake\Cache\Engine\FileEngine.php, line 168]
2014-05-12 22:13:29 Error: [FatalErrorException] Maximum execution time of 30 seconds exceeded
Request URL: /shop/products
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Maximum executi...', 'C:\xampp\htdocs...', 168)
#1 [internal function]: ErrorHandler::handleError(1, 'Maximum executi...', 'C:\xampp\htdocs...', 168, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Maximum executi...', 'C:\xampp\htdocs...', 168, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:14:52 Error: [MissingViewException] View file "C:\xampp\htdocs\shop\View\Home\home.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\shop\\View\\Home\\home.ctp',
)
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(463): View->_getViewFileName('home')
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(954): View->render('home', NULL)
#2 C:\xampp\htdocs\shop\Controller\HomeController.php(60): Controller->render('home')
#3 [internal function]: HomeController->display('home')
#4 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(HomeController), Array)
#5 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(185): Controller->invokeAction(Object(CakeRequest))
#6 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(HomeController), Object(CakeRequest), Object(CakeResponse))
#7 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2014-05-14 01:25:46 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 1]
2014-05-14 01:25:46 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:29:36 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 1]
2014-05-14 01:29:36 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:29:52 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 1]
2014-05-14 01:29:52 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:30:05 Error: Fatal Error (1): Call to a member function css() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 45]
2014-05-14 01:30:05 Error: [FatalErrorException] Call to a member function css() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:30:32 Error: [MissingViewException] View file "C:\xampp\htdocs\shop\View\Home\display.ctp" is missing.
Exception Attributes: array (
  'file' => 'C:\\xampp\\htdocs\\shop\\View\\Home\\display.ctp',
)
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\View\View.php(463): View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\cake\lib\Cake\Controller\Controller.php(954): View->render(NULL, NULL)
#2 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(192): Controller->render()
#3 C:\xampp\htdocs\cake\lib\Cake\Routing\Dispatcher.php(160): Dispatcher->_invoke(Object(HomeController), Object(CakeRequest), Object(CakeResponse))
#4 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-05-14 01:31:10 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 1]
2014-05-14 01:31:10 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:33:28 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 1]
2014-05-14 01:33:28 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:33:30 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 1]
2014-05-14 01:33:30 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 1, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:34:23 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\Controller\HomeController.php, line 40]
2014-05-14 01:34:23 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 40)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 40, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 40, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:34:25 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\Controller\HomeController.php, line 40]
2014-05-14 01:34:25 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 40)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 40, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 40, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:36:13 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\Controller\HomeController.php, line 41]
2014-05-14 01:36:13 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 41)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 41, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 41, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:36:16 Error: Fatal Error (1): Call to a member function charset() on a non-object in [C:\xampp\htdocs\shop\Controller\HomeController.php, line 41]
2014-05-14 01:36:16 Error: [FatalErrorException] Call to a member function charset() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 41)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 41, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 41, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:39:28 Error: Fatal Error (1): Call to a member function css() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 45]
2014-05-14 01:39:28 Error: [FatalErrorException] Call to a member function css() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:42:04 Error: Fatal Error (1): Call to a member function css() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 45]
2014-05-14 01:42:04 Error: [FatalErrorException] Call to a member function css() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:42:45 Error: Fatal Error (1): Call to a member function css() on a non-object in [C:\xampp\htdocs\shop\View\Home\home.ctp, line 45]
2014-05-14 01:42:45 Error: [FatalErrorException] Call to a member function css() on a non-object
Request URL: /shop/
Stack Trace:
#0 C:\xampp\htdocs\cake\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#2 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(929): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\xampp\htdocs...', 45, Array)
#3 C:\xampp\htdocs\cake\lib\Cake\Core\App.php(902): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-05-14 01:42:45 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /shop/css/assets/plugins/font-awesome/css/font-awesome.min.css
Stack Trace:
#0 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-05-14 01:42:45 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /shop/css/assets/plugins/bootstrap/css/bootstrap.css
Stack Trace:
#0 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-05-14 01:44:25 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /shop/css/assets/plugins/font-awesome/css/font-awesome.min.css
Stack Trace:
#0 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-05-14 01:44:25 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /shop/css/assets/plugins/bootstrap/css/bootstrap.css
Stack Trace:
#0 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-05-14 02:13:43 Error: [MissingControllerException] Controller class Product-list.htmlController could not be found.
Exception Attributes: array (
  'class' => 'Product-list.htmlController',
  'plugin' => NULL,
)
Request URL: /shop/product-list.html
Stack Trace:
#0 C:\xampp\htdocs\shop\webroot\index.php(99): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
